application.rb

```diff
@@ -19,3 +19,7 @@
 ].each do |pattern|
   Dir.glob(Config.root.join(*pattern)).each { |file| require file }
 end
+
+::ActiveRecord::Base.schema_format = :sql
+# XXX: Update created_at and updated_at in PostgreSQL
+::Measurement.record_timestamps = false
```

Comment:
Seems like this should go into the measurement class directly. Or at least the same file.

Unclear on what the XXX signifies.